### PR TITLE
Add ES modules, classes.

### PR DESCRIPTION
**Pull request by mbostock**
I modeled this after our earlier work on Delaunator, but I’m happy to make any changes if you’d prefer a different approach.

### CODE REVIEW

**Review by mourner — APPROVED** [review]
Looks great! Also needs a Travis config update (node 4 -> 6) to fix the build.

File: package.json [hunk 1]
```diff
@@ -26,9 +38,19 @@
   "devDependencies": {
     "eslint": "^4.8.0",
     "eslint-config-mourner": "^2.0.1",
+    "esm": "^3.0.8",
+    "rollup": "^0.57.1",
+    "rollup-plugin-buble": "^0.19.2",
+    "rollup-plugin-uglify": "^3.0.0",
```
Comment (mourner) [inline package.json]: Also let's use newer versions here.

**Comment by mbostock** [comment]
I’ve updated the Node version for Travis to 6 as requested, and updated Rollup to 0.64. It looks like rollup-plugin-uglify no longer supports transpiling ES6+ syntax, and while we don’t currently need that (since we aren’t publishing a minified ES module), I figured we might as well switch to rollup-plugin-terser.

**Comment by mourner** [comment]
Thanks! Published as v2.0.0.

**Pull request merged**